src/main/java/uk/gov/companieshouse/api/accounts/transformer/PreviousPeriodTransformer.java

```diff
@@ -30,6 +32,14 @@ public PreviousPeriodEntity transform(PreviousPeriod entity) {
                     .copyProperties(entity.getBalanceSheet().getFixedAssets(), fixedAssetsEntity);
                 balanceSheetEntity.setFixedAssets(fixedAssetsEntity);
             }
+
+            if(entity.getBalanceSheet().getCurrentAssets() != null){
```

Comment:
Can we add spaces before and after brackets please: 
`if (entity.getBalanceSheet().getCurrentAssets() != null) {`